### PR TITLE
Update extraRpcs.js

### PR DESCRIPTION
**Pull request by geopayme**

### CODE REVIEW

File: constants/extraRpcs.js [hunk 1]
```diff
@@ -99,6 +99,11 @@ export default {
                 url: "https://eth-mainnet-public.unifra.io",
                 tracking: "unspecified"
             },
+            {
+                url: "https://eth.node.ibbt.io/265011c66203eb5375fe0c96d72281a4822b05a2/",
+                tracking: "none",
```
[the remark this answers is not in the document]
Comment (geopayme) [reply constants/extraRpcs.js:104]: Please find that our policy regarding NODE RPC services has been updated to reflect no data collection from our NODE services.

File: constants/extraRpcs.js [hunk 2]
```diff
@@ -99,6 +99,11 @@ export default {
                 url: "https://eth-mainnet-public.unifra.io",
                 tracking: "unspecified"
             },
+            {
+                url: "https://eth.node.ibbt.io/265011c66203eb5375fe0c96d72281a4822b05a2/",
+                tracking: "none",
+                trackingDetails: privacyStatement.ibbt,
```
[the remark this answers is not in the document]
Comment (geopayme) [reply constants/extraRpcs.js:105]: the policy has been updated to reflect the zero collection from our iBBT RPC Node Services

**Comment by geopayme** [comment]
I see ok

**Comment by realdealshaman** [comment]
@geopayme Hi, can you add the portion of privacy policy that states no user data is collected. This can be added to the top of the file , there you will see others that you can use as an example. Once this is added, we can merge your PR. Thanks

**Comment by geopayme** [comment]
Ok, I will do that.

*"I don't need to know everything; I just need to know where to find it." *
*A. Einstein.*

If you need to reach your people fast, there's no better way than with
"mobile." Mobile is direct, accurate, and personal.

This email may contain confidential and privileged information. If you are
not the intended recipient (or have received this email by mistake), please
notify the sender immediately and destroy this email. Any unauthorized
copying, disclosure, or distribution of the material in this email is
strictly prohibited.

Email transmission security and error-free status cannot be guaranteed as
information could be intercepted, corrupted, destroyed, delayed,
incomplete, or contain viruses. Therefore, the sender does not accept
liability for any errors or omissions in this message that may arise due to
email transmission.

**Pull request closed without merging**